_projects/SP19-VRTrapped.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: project
+title: VR Trapped
+desc: |
+ A Virtual Reality Escape Room for Oculus Rift
+dl: https://drive.google.com/open?id=1dIKXqPt3RJlLoFk3ttvtqeGxL_d6g3JJ
```

Comment:
Make sure this link is accessible to anyone with the link, and make sure it contains the Data folder.